Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -109,11 +112,17 @@ struct ScheduleTransferSummaryData: ScheduleTransferSectionData {
             let grossTransferAmount = transferAmountFormattedDouble + feeAmountFormattedDouble
 
             rows.append((title: "transfer_amount_confirmation".localized(),
-                         value: String(grossTransferAmount).format(with: destinationCurrency)))
+                         value: String(format: "%@ %@",
+                                       String(grossTransferAmount)
+                            .format(with: destinationCurrency), destinationCurrency)))
             rows.append((title: "transfer_fee_confirmation".localized(),
-                         value: feeAmount.format(with: destinationCurrency)))
+                         value: String(format: "%@ %@",
+                                       feeAmount.format(with: destinationCurrency),
+                                       destinationCurrency)))
             rows.append((title: "transfer_net_amount_confirmation".localized(),
-                         value: destinationAmountFormattedString))
+                         value: String(format: "%@ %@",
+                                       destinationAmountFormattedString,
```

Comment:
No need to format this as per my previous review comment, so use destinationAmount instead